packages/mobile/e2e/src/utils/utils.js

```diff
@@ -251,3 +251,11 @@ export async function scrollIntoView(scrollTo, scrollIn, speed = 350, direction
       .scroll(speed, direction)
   } catch {}
 }
+
+export async function getDeviceModel() {
```

Comment:
I thought there was a detox util that does this but I might be mistaking it for something else ...

Reply:
There is a `device.getPlatform()`, but no model information is provided. `device.getUiDevice()` is Android only at present.

https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md